docs/UserGuide.md

```diff
@@ -332,7 +333,7 @@ Example:
 
 Clears all events within a specified time range.
 
-Format: `clearEvents ts/START_DATE_TIME te/END_DATE_TIME c/CONFIRMATION`
+Format: `clearEvents ts/START_DATE_TIME te/END_DATE_TIME [c/CONFIRMATION]`
```

Comment:
Would it be better to specify below that if users enter the command, they will be prompted to confirm the event since the clearing cannot be undone?